Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -47,6 +47,14 @@ public class SessionSearch
         public Epic.OnlineServices.Sessions.SessionSearch SearchHandle;
         private Dictionary<Session, SessionDetails> SearchResults = new Dictionary<Session, SessionDetails>();
 
+        /// <summary>
+        /// Action to run when search results are successfully retrieved.
+        /// Set in <see cref="SetNewSearch(Epic.OnlineServices.Sessions.SessionSearch, Action{SessionSearch})"/>,
+        /// unset in <see cref="Release"/>, when a new search is set without the argument,
+        /// or after <see cref="OnSearchResultReceived(Dictionary{Session, SessionDetails})"/> is called.
+        /// </summary>
+        private Action<SessionSearch> runOnSearchResultReceived { get; set; }
```

Comment:
I'm of the opinion that all properties should be capitalized, regardless of access modifiers used. For field _members_ I would say private ones should start with an underscore and be camel case. In summary: I believe that this should be capitalized.

Reply:
Agreeable! I'll go set that up now.